specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/Servers.json

```diff
@@ -502,13 +540,17 @@
             "modelAsString": true
           }
         },
-        "encryptionIdentityId": {
-          "description": "The resource id of a user assigned identity to be used to access the customer managed keyvault.",
+        "primaryUserAssignedIdentityId": {
```

Comment:
Is there strong business reason to  rename this properties?  it will cause SDK breaking change in most of case

Reply:
> > 2020-11-01-preview version already released, these change may need to new API version. @ericshape to suggest
> 
> @akning-ms This change is in the Server and Managed Instance API and has no live customers. Most of the changes were missed in the API due to error but they must go in the 2020-11-01-preview API version release.
> 
> 1. Changes in MI api are similar to Server API but were not cherry picked for some reason while releasing the swagger.
> 2. A minor update has been made to the Server API due to JEDI requirements.
> 3. This is not being consumed by any live SDKs.

You must hold .Net release first.  @ericshape , and curious how do you check whether the managedInstance API no any live SDK consume?

Reply:
I checked with my team and it seems like NET SDK has not been released yet?

> Is there strong business reason to rename this properties? it will cause SDK breaking change in most of case
Business reason from my PM:
The property rename is needed to support User-Assigned Managed Identities for all SQL features which is an important Walmart M9 ask. This property is not specific to encryption only but is rather intended to be used as the default user-assigned identity of the server, so that any SQL feature that wishes to onboard to user-assigned managed identities can readily consume this property.

Reply:
The two specs files which Shobhit modified in this PR are:
- 2020-11-01-preview/ManagedInstances.json
- 2020-11-01-preview/Servers.json

And @strehan1993 did not update any SDK version tag. As the v4 SDK tag (current SDK release) in: https://github.com/Azure/azure-rest-api-specs/tree/master/specification/sql/resource-manager#tag-package-composite-v4
ManagedInstances.json is using `- Microsoft.Sql/preview/2020-02-02-preview/ManagedInstances.json`
Servers.json is using `- Microsoft.Sql/preview/2019-06-01-preview/servers.json`

In addition, we did not release any SDK contains the latest ManagedInstances.json and Servers.json. Thus, we think the latest ManagedInstance and Server specs don't have live customers until now since we publish 2020-11-01 last Wednesday.

Reply:
If this 2 files in PR is not in current v4 SDK tag, consider .Net is already released for this version in master. Not sure what is the reason to put this feature in 2021-11-01-preview? can we put them in next API version? normally we treat merge into master is a release, especially after SDK release. otherwise it will cause lots of confusion as there is  2 different 2021-11-01-preview version API.

Reply:
and also rename a property may have big impact for SDK/CLI for existing customer. can it be better to add a new one primaryUserAssignedIdentityId and keep encryptionIdentityId to keep compatible with previous version? and put some deprecate info in description

Reply:
As mentioned the property was added 3 days ago, no one is using it. 
2020-11-01-preview was released incorrectly with missing functionality, and we need to update it to reflect the right swagger.